frontend/yarn.lock

```diff
@@ -0,0 +1,5890 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
`yarn.lock` is just like the `Gemfile.lock`. It should be version controlled.


> All yarn.lock files should be checked into source control

https://yarnpkg.com/lang/en/docs/yarn-lock/